web/src/connectors/edit-job-connector/edit-job-connector.tsx

```diff
@@ -78,7 +78,7 @@ const EditJobConnector: FC<EditJobConnectorProps> = ({
     showNoExtractionTab
 }) => {
     const getMetadata = (state: JobValues) => {
-        const { jobType, validationType, annotators_validators, pipeline, validators, categories } =
+        const { jobType, validationType, annotators_validators, pipeline, validators,annotators, categories } =
```

Comment:
Where is this annotators value being used?

Reply:
I forgot to remove unnecessary variables, fixed.